src/main/java/edu/indiana/dlib/amppd/controller/MediaController.java

```diff
@@ -58,14 +51,14 @@
     }
 
 	/**
-	 * Serve the output file of the given dashboardResult by redirecting the request to the AMPPD UI Apache server.
-	 * @param id ID of the given dashboardResult
+	 * Serve the output file of the given workflowResult by redirecting the request to the AMPPD UI Apache server.
+	 * @param id ID of the given workflowResult
 	 * @return the content of the output file
 	 */
-	@GetMapping("/dashboard/{id}/output")
-	public ResponseEntity<Object> serveDashboardOutput(@PathVariable("id") Long id) {		
-    	log.info("Serving output for dashboardResult ID " + id);
-    	String url = mediaService.getDashboardOutputSymlinkUrl(id);
+	@GetMapping("/workflow-result/{id}/output")
```

Comment:
below workflow-results/ is used for the path, lets use the plural here too to be consistent

Reply:
Good catch!